lido-landing/faq.md

```diff
@@ -0,0 +1,390 @@
+---
+categories:
+  - name: Introduction
+    q&a:
+      - question: What is Lido?
+        answer: Lido is the name of a family of open-source peer-to-system software
+          tools deployed and functioning on the Ethereum, Solana, and Polygon
+          blockchain networks. The software enables users to mint transferable
+          utility tokens, which receive rewards linked to the related validation
+          activities of writing data to the blockchain, while the tokens can be
+          used in other on-chain activities.
+      - answer: While each network works differently, generally, the Lido protocols
+          batch user tokens to stake with validators and route the staking
+          packages to network staking contracts. Users mint amounts of stTokens
+          which correspond to the amount of tokens sent as stake and they
+          receive staking rewards. When they unstake, they burn the stToken to
+          initiate the network-specific withdrawal process to withdraw the
+          balance of stake and rewards.
+        question: How does Lido work?
```

Comment:
order (answer, then question) is strange, any idea how it happened?

Reply:
Such behavior occurs if you first write the answer, and then the question... this is the internal behavior of the CMS... it does not affect the file operation in any way.
Or is the sequence in the file important to us for other reasons?

Reply:
just worried that something gone wrong